packages/server/next/app/components/ui/chat/chat-section.tsx

```diff
@@ -33,7 +33,7 @@ export default function ChatSection() {
   return (
     <>
       <div className="flex h-screen w-screen flex-col overflow-hidden">
-        <ChatHeader />
+        <Header appTitle={getConfig("APP_TITLE")} />
```

Comment:
we can remove APP_TITLE from config if we are only using it for the header as the user can change it directly in header.tsx

Just add it to the docs